dll/win32/syssetup/lang/it-IT.rc

```diff
@@ -133,20 +133,20 @@ END
 
 IDD_PS2MOUSEPROPERTIES DIALOGEX 0, 0, 252, 218
 STYLE WS_CHILD | WS_VISIBLE | WS_CAPTION
-CAPTION "Advanced Settings"
+CAPTION "Impostazioni avanzate"
 FONT 8, "MS Shell Dlg"
 BEGIN
     RTEXT "&Sample Rate:", -1, 27, 35, 90, 8
-    LTEXT "reports per second", -1, 169, 35, 76, 8
+    LTEXT "report per secondo", -1, 169, 35, 76, 8
     COMBOBOX IDC_PS2MOUSESAMPLERATE, 124, 33, 42, 41, CBS_DROPDOWNLIST | WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_VSCROLL
     RTEXT "&Wheel Detection:", -1, 27, 55, 90, 8
```

Comment:
?

Reply:
That was not translated before

Reply:
To clarify @HBelusca's doubt. In Italian, the words borrowed from English are never expressed in plural with `s` but rather we use articles to define the word as plural or singular.

For example, we would never say `I leaders` but `I leader` because `I` is a masculine plural form. That goes the same for other borrowed English words and `report`.

Reply:
Actually I'm quite shocked by the number of words you say italian borrows from english, when these are trivial and translations should exist even before (in this context) computer mice existed.

Reply:
There are a transaltion of 'report' but on IT context we use a lot of english terms.

Reply:
I think French is the exception ;)  German similarly has many IT terms that aren't usually translated, e.g. computer, input/output, update, desktop, software/hardware